dlrover/python/elastic_agent/torch/training.py

```diff
@@ -379,6 +381,8 @@ def __init__(
         start_method="spawn",
         exit_barrier_timeout: float = 300,
         log_dir: Optional[str] = None,
+        training_log_file: str = "",
```

Comment:
Considering such variables will be added in the future, it is necessary to define a unified POJO to encapsulate these variables.